Quick_Deploy/vLLM/client.py

```diff
@@ -35,109 +35,124 @@
+
+    async def stream_infer(self, prompts, sampling_parameters):
         try:
             # Start streaming
-            response_iterator = triton_client.stream_infer(
-                inputs_iterator=async_request_iterator(),
-                stream_timeout=FLAGS.stream_timeout,
+            response_iterator = self._client.stream_infer(
+                inputs_iterator=self.async_request_iterator(prompts, sampling_parameters),
+                stream_timeout=self._flags.stream_timeout,
             )
-            # Read response from the stream
             async for response in response_iterator:
-                result, error = response
-                if error:
-                    print(f"Encountered error while processing: {error}")
-                else:
-                    output = result.as_numpy("TEXT")
-                    for i in output:
-                        results_dict[result.get_response().id].append(i)
-
+                yield response
         except InferenceServerException as error:
             print(error)
             sys.exit(1)
 
-    with open(FLAGS.results_file, "w") as file:
-        for id in results_dict.keys():
-            for result in results_dict[id]:
-                file.write(result.decode("utf-8"))
-                file.write("\n")
-            file.write("\n=========\n\n")
-        print(f"Storing results into `{FLAGS.results_file}`...")
+    async def process_stream(self, prompts, sampling_parameters):
+        # Clear results in between process_stream calls
+        self.results_dict = []
+        
+        # Read response from the stream
+        async for response in self.stream_infer(prompts, sampling_parameters):
+            result, error = response
+            if error:
+                print(f"Encountered error while processing: {error}")
+            else:
+                output = result.as_numpy("TEXT")
+                for i in output:
+                    self._results_dict[result.get_response().id].append(i)
+
+    async def run(self):
+        sampling_parameters = {"temperature": "0.1", "top_p": "0.95"}
+        stream = self._flags.streaming_mode
+        with open(self._flags.input_prompts, "r") as file:
+            print(f"Loading inputs from `{self._flags.input_prompts}`...")
+            prompts = file.readlines()
+
+        await self.process_stream(prompts, sampling_parameters)
+
+        with open(self._flags.results_file, "w") as file:
+            for id in self._results_dict.keys():
+                for result in self._results_dict[id]:
+                    file.write(result.decode("utf-8"))
+                    file.write("\n")
+                file.write("\n=========\n\n")
+            print(f"Storing results into `{self._flags.results_file}`...")
+
+        if self._flags.verbose:
+            print(f"\nContents of `{self._flags.results_file}` ===>")
+            system(f"cat {self._flags.results_file}")
```

Comment:
I'd rather use pure python than call out to bash. I think you can just do `with open(...) as f: print(f.read())`

Reply:
Great thinking, thanks for the suggestion! Done.
![image](https://github.com/triton-inference-server/tutorials/assets/58150256/58b4e5e3-3b60-411f-94ee-61fe2ca2287f)